front/cypress/e2e/admin/pages_and_menu/update_homepage_hero_banner_content.cy.ts

```diff
@@ -94,6 +94,7 @@ describe('Admin: update Hero Banner content', () => {
     cy.get('[data-cy="e2e-overlay-toggle"]').should('exist');
     cy.get('.e2e-submit-wrapper-button').click();
     cy.wait('@saveHomePage');
+    cy.wait(2000);
```

Comment:
In some rare cases, the only way I can make something less flaky is to add a wait. The e2e tests passed on ci when I tested using this branch, so hopefully this does the trick :) I was never able to reproduce this locally though either.